imports/ui/components/CardList.jsx

```diff
@@ -24,7 +66,9 @@ class CardList extends Component {
           placeholder="Find Item"
           onChange={true}
         />
-        {/* TODO: add filter */}
+        {/* Sort by price and rating */}
+        <p onClick={this.sortByPricePressed}>Sort by price</p>
+        <p onClick={this.sortByLatestPressed}>Sort by latest</p>
```

Comment:
I didn't want to do a db call every time we need a sort, because that would be expensive. Not that crazy because our project is small, but I think FE handling is more normal.

Reply:
Gotcha, are we okay to keep this sorted or should we leave all sorting to FE?